### ISSUE TITLE
Invalid import statement leads to goroutine leak

### ISSUE THREAD

**Issue by cyrixsimon** [exasol/exasol-driver-go#79]
Hi,

the execution of an invalid import statement, that causes a database error, leads to a goroutine leak. 

I have analyzed the problem and could clarify why this leak occurs. In the exec method the errs.Wait() will never return if the import statement causes an error. 
```go
func (c *connection) exec(ctx context.Context, query string, args []driver.Value) (driver.Result, error) {
...
	if isImportQuery(query) {
...
		errs.Go(func() error { return c.uploadFiles(p, originalQuery) })
	}
...
	err := errs.Wait()
	close(result)

	if err != nil {
		return nil, err
	}

	return <-result, nil
}
```
This is because the import statement is handled through two goroutines, one for reading the file and another for handling the sql statement. 
```go
func (c *connection) exec(ctx context.Context, query string, args []driver.Value) (driver.Result, error) {
...
errs.Go(func() error { return c.uploadFiles(p, originalQuery) })
...
errs.Go(c.executeSimpleWrapper(errctx, query, result))
...
```
Both goroutines increase the counter of the waitgroup (errs.wg). Whenever an error is returned from the goroutine which handels the sql statement the first goroutine never recognizes this and therefore keeps waiting. So overall the additional goroutine with import statements which handels the file processing does not notice when an error occurs and therefore never returns.

**Comment by kaklakariada** [comment]
We added this to our backlog, but it will take about four weeks until we can work on this.
Thank you for your patience!

**Comment by kaklakariada** [comment]
@cyrixsimon
We just released version [v0.4.6](https://github.com/exasol/exasol-driver-go/releases/tag/v0.4.6) with a bugfix for this.
